### PR TITLE
Challenges Refactor

### PR DESCRIPTION
**Pull request by goalves**
Better but not good enough yet.

### CODE REVIEW

File: challenge3.md [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
-The design is up to you!
+The application to be developed needs to have the basic four features:
+
+- List issues using the Github API.
+- Sort issues either by id or state.
+- Favorite issues.
+- Navigate to a different screen that will have all the issues that were favorited.
 
-Please write the code as if it would go to production right after you commit it!
+Please note that the intent of any challenge is not to 'solve the problem', but showcase good practices while solving the problem.
```
Comment (hoarf) [inline challenge3.md]: - not *only* to solve the problem

remove the quotes

File: README.md [hunk 2]
```diff
@@ -1,17 +1,45 @@
...
+    - Standards conformity
+    - Branching, Complexity and Data Flow
+2. Semantic Analysis:
+    - Deliverability
+    - Testability
+    - Coherence
+    - Cohesion
+    - Coupling
+
+Everything you do will be evaluated.
```
Comment (hoarf) [inline README.md]: this contradicts the previous statement.. either the evaluation is specific or general

File: README.md [hunk 3]
```diff
@@ -1,17 +1,45 @@
...
+Keep in mind that each challenge is for a different development area (mobile, backend, frontend) and your knowledge on the stack, tools, frameworks and everything related to the specific area will also be evaluated.
+
+**Please do not forget to write a good README file with instructions to run and test your code :).**
+
+## Important Notes
+
+- If you have any questions, reach out to us and we will gladly help, you are not going to be penalized by this.
+- Take as much time as needed and **communicate as much as necessary**.
+- The challenges are not about something that 'solves the problem', but something that solves the problem while showcasing good practices and the best of your abilities.
```
Comment (hoarf) [inline README.md]: quotes are note necessary. I'd rephrase this to The challenge is not only to solve the problem but showcase how much you know about good programming practices.

File: README.md [hunk 4]
```diff
@@ -1,17 +1,45 @@
...
+- Write the best code that you can, in the language you are most confortable with. Do not try a fancy language just to showcase how fast can you learn something.
```
Comment (hoarf) [inline README.md:34]: Add: You do not get bonus points for using Elixir or Elm if they are not languages you are comfortable with.

File: challenge3.md [hunk 5]
```diff
@@ -1,26 +1,31 @@
...
 
-Feel free to take as much time as you want and send us the code when you feel conformable with it!
+Feel free to take as much time as you want and send us the code when you feel comfortable with.
 
-## FAQ
+## Tips
 
-* Github has no API for making issues as favorites. How can I implement it?
+- You are free to use CSS pre-processors if you want to.
+- You can decide whether you want to manage state or not in the application.
```
Comment (hoarf) [inline challenge3.md]: Don't know what you mean by this

**Review by hoarf — APPROVED** [review]

File: challenge1.md [hunk 6]
```diff
@@ -1,24 +1,23 @@
-# Deepx's Mobile Challenge
+# DeepX Mobile Challenge
 
 ## Description
-Develop a mobile application that consumes the github REST api and lists all the repositories for the `BearchInc` organization. The app consists of fetching, caching and displaying data. The app should work offline.
 
+Develop a mobile application that consumes the github REST api and lists all the repositories for the `BearchInc` organization. The app consists of fetching, caching and displaying data. The app should work offline.
```
Comment (amarantedaniel) [inline challenge1.md]: BearchInc doesn't exist anymore

File: challenge3.md (hunk 5 above)
Comment (goalves) [reply challenge3.md]: Both removed, I added to make sure the candidate understand that this is a web application.

File: README.md (hunk 2 above)
Comment (goalves) [reply README.md]: leftover actually heh

File: challenge3.md (hunk 1 above)
Comment (goalves) [reply challenge3.md]: Fixed, removed the quotes and added __only__
